open_lm/model_configs/open_lm_test_tiny.json

```diff
@@ -0,0 +1,9 @@
+{
+    "hidden_dim": 96,
+    "n_layers": 2,
+    "n_heads": 2,
+    "seq_len": 2048,
+    "vocab_size": 50432,
```

Comment:
being a bit tedious here so feel free to ignore but couldn't we go down to some other setting for testing - like character level tokenization (would make vocab_size much smaller) and shorter ctx len?

Right now even with hidden dim 96 you will have at least `96 * 50432 = 4841472` embedding params and this last matmul for the output probs will be slow

Reply:
this is obv fine for now if it works

Reply:
not tedious at all, makes sense, especially for this test. made everything smaller